api/src/certification/enrolment/application/usecases/checkUserIsCandidate.js

```diff
@@ -0,0 +1,16 @@
+import * as candidateRepository from '../../infrastructure/repositories/candidate-repository.js';
+
+const execute = async ({ userId, certificationCandidateId, dependencies = { candidateRepository } }) => {
+  const isUserCertificationCandidate = dependencies.candidateRepository.isUserCertificationCandidate({
+    certificationCandidateId,
+    userId,
+  });
+
+  if (!isUserCertificationCandidate) {
+    return false;
+  }
+
+  return true;
```

Comment:
Il manque un await ligne 4. Le repository semble déjà renvoyer un boolean en retour.

```suggestion
  return dependencies.candidateRepository.isUserCertificationCandidate({
    certificationCandidateId,
    userId,
  });
```